.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
     - <<: *node_container
       name: 'Node - build, type-check, and end-to-end tests'
       script:
-        - yarn build
+        - yarn build --verbose
```

Comment:
Travis keeps timing out because the build takes >10m with no output.  Making it verbose works around Travis' behaviour 🤷🏻‍♂️

Reply:
tbh it also works around my behaviour, I get a gut feeling that it's broken every time since we changed it from the old setup :P